src/Discovery/Discovery.tsx

```diff
@@ -298,11 +303,17 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
     return indexArr;
   };
 
-  const [selectedSearchableTextFields, setSelectedSearchableTextFields] = useState([] as string[]);
+  const { searchableTextFields = [], searchableAndSelectableTextFields = {} } = config?.features?.search?.searchBar || {};
+  const allSearchableFields = [...searchableTextFields, ...Object.values(searchableAndSelectableTextFields)] as string[];
+  const [selectedSearchableTextFields, setSelectedSearchableTextFields] = useState(allSearchableFields);
```

Comment:
Do we want to change the name of the variable ```selectedSearchableTextFields``` if it no longer represents only the user's selections? It looks like it is being initialized here as allSearchableFields instead of the users selections at page load (an empty array before this update).